### PR TITLE
Add undeploy.yaml playbook

### PR DESCRIPTION
**Pull request by marianoleonardo**
* **Please check if the PR fulfills these requirements**
  * [ ] Tests for the changes have been added (for bug fixes / features)
  * [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Feature

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**
Now it is possible to undeploy dojot using a playbook

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)
[DEVOPS-362](https://jira.cpqd.com.br/browse/DEVOPS-362)

* **Other information**:

### CODE REVIEW

**Review by gabrielifg — APPROVED** [review]
Bom trabalho 👏👏👏👏👏

File: roles/undeploy/dojot-volumes/directory/database/tasks/main.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+- name: Remove Database volume directory
+  become: yes
+  file:
+    path: "{{ dojot_volume_directory }}/{{item.path}}"
+    state: absent
+  loop:
+    - { path: 'influxdb' }
+    - { path: 'influxdb-chronograf' }
+  when: (dojot_storage_class_name == "local-storage")
```
Comment (danielapetito) [inline roles/undeploy/dojot-volumes/directory/database/tasks/main.yaml:11]: O storage_class_name não deveria tratar local-storage e nfs ?

File: roles/undeploy/dojot-volumes/directory/dojot/tasks/main.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  file:
+    path: "{{ dojot_volume_directory }}/{{item.path}}"
+    state: absent
+  loop:
+    - { path: 'kong' }
+    - { path: 'minio' }
+    - { path: 'minio-files' }
+    - { path: 'mongodb' }
+    - { path: 'postgres' }
+  when: (dojot_storage_class_name == "local-storage")
```
Comment (danielapetito) [inline roles/undeploy/dojot-volumes/directory/dojot/tasks/main.yaml:14]: O storage_class_name não deveria tratar local-storage e nfs ?

File: roles/undeploy/dojot-volumes/directory/kafka/tasks/main.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+- name: Remove Kafka volume directory
+  become: yes
+  file:
+    path: "{{ dojot_volume_directory }}/{{item.path}}"
+    state: absent
+  loop:
+    - { path: 'kafka' }
+    - { path: 'kafka_ws' }
+    - { path: 'zookeeper' }
+  when: (dojot_storage_class_name == "local-storage")
```
Comment (danielapetito) [inline roles/undeploy/dojot-volumes/directory/kafka/tasks/main.yaml:12]: O storage_class_name não deveria tratar local-storage e nfs ?

File: roles/undeploy/dojot-volumes/directory/x509/tasks/main.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+- name: Remove X509 volume directory
+  become: yes
+  file:
+    path: "{{ dojot_volume_directory }}/{{item.path}}"
+    state: absent
+  loop:
+    - { path: 'x509-identity-mgmt' }
+    - { path: 'ejbca' }
+    - { path: 'certificate_acl' }
+  when: (dojot_storage_class_name == "local-storage")
```
Comment (danielapetito) [inline roles/undeploy/dojot-volumes/directory/x509/tasks/main.yaml:12]: O storage_class_name não deveria tratar local-storage e nfs ?

File: roles/undeploy/monitoring-volumes/directory/tasks/main.yaml [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+- name: Remove Monitoring volume directory
+  become: yes
+  file:
+    path: "{{ dojot_volume_directory }}/{{item.path}}"
+    state: absent
+  loop:
+    - { path: 'prometheus' }
+    - { path: 'grafana' }
+    - { path: 'loki' }
+  when: (dojot_storage_class_name == "local-storage")
```
Comment (danielapetito) [inline roles/undeploy/monitoring-volumes/directory/tasks/main.yaml:12]: O storage_class_name não deveria tratar local-storage e nfs ?

**Review by danielapetito — APPROVED** [review]

**Review by gabrielifg — APPROVED** [review]
GG!!

**Pull request merged**